railties/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Support prerelease rubies in Gemfile template if RubyGems version is 3.3.16 or higher.
```

Comment:
```suggestion
*   Support prerelease rubies in Gemfile template if RubyGems version is 3.3.13 or higher.
```

Should this be the same as the version checks?

Reply:
Thank you pointing this out.
`3.3.13` is correct. so just changing the changelog entry is fine.